symfony/messenger/6.0/manifest.json

```diff
@@ -4,9 +4,9 @@
     },
     "env": {
         "#1": "Choose one of the transports below",
-        "#2": "MESSENGER_TRANSPORT_DSN=doctrine://default",
-        "#3": "MESSENGER_TRANSPORT_DSN=amqp://guest:guest@localhost:5672/%2f/messages",
-        "#4": "MESSENGER_TRANSPORT_DSN=redis://localhost:6379/messages"
+        "#2": "MESSENGER_TRANSPORT_DSN=amqp://guest:guest@localhost:5672/%2f/messages",
+        "#3": "MESSENGER_TRANSPORT_DSN=redis://localhost:6379/messages",
+        "MESSENGER_TRANSPORT_DSN": "doctrine://default"
```

Comment:
`?auto_setup=0`?

Reply:
because we prefer using migrations to define the table for messages

Reply:
But you said you want something that works out of the box. I'm a bit lost

Reply:
There is a reason to the current way, but I also get your point. I don't have the final solution :)

Reply:
Let's do it step by step. This PR with `?auto_setup=0` is already a step in the right direction IMO.
We need to make migrations more front and center I suppose.
Many frameworks do that already (Rails, Laravel, ...).
Maybe we can add a post install instruction?

Reply:
Done 👍🏼